src/mococrw/openssl_wrap.h

```diff
@@ -1339,6 +1341,19 @@ enum class ellipticCurveNid
     Ed25519 = NID_ED25519
 };
 
+enum class ellipticCurvePointConversionForm
```

Comment:
I haven't checked whether there's a precedent for this, but don't we usually capitalize types, i.e. should we make this `EllipticCurvePointConversionForm`?

Reply:
Won't change.

Reply:
No will change! This is part of the public interface! Type names are capitilized.